LuisWalterArredondoAmarillas/P05/P05.ipynb

```diff
@@ -0,0 +1,575 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    word_probabilities_spa_tokenized_types = {word: count / len(spa_bible_words) for word, count in spa_bible_tokenized_types.items()}
</pre><p>Creo que el problema es que estás dividiendo por <code>spa_bible_words</code> en lugar del tamaño del corpus tokenizado</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/umoqnier/cl-2024-1-lab/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>